### PR TITLE
docs(PULL_REQUEST_TEMPLATE.md): fix broken links in PR template

### PR DESCRIPTION
**Pull request by frederikprijck**
fix broken links in PR template

### CODE REVIEW

**Comment by frederikprijck** [comment]
I'm not sure if hardcoding `blob/master` is a good idea. But I prefer a working link instead of a broken. 😬

**Review by gkalpak — APPROVED** [review]

**Comment by gkalpak** [comment]
Can you change the commit message to `docs(PULL_REQUEST_TEMPLATE.md): ...`?

**Comment by frederikprijck** [comment]
@gkalpak Updated! :)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -16,8 +16,8 @@
 
 
 **Please check if the PR fulfills these requirements**
-- [ ] The commit message follows our [guidelines](../DEVELOPERS.md#commits)
```
Comment (Narretz) [inline .github/PULL_REQUEST_TEMPLATE.md]: Can you please change this and the other link below to be relative? `../blob/master/DEVELOPERS.md#commits` This way, they work the same in forks.

**Review by Narretz — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (frederikprijck) [reply .github/PULL_REQUEST_TEMPLATE.md]: Mmm this doesn't work in every situation, using an absolute url does.

I did the following:
- Edit the template file using github UI
- Update the URL to reflect ur changes
- Preview the file
- Click the link to test if it works and see it won't work.

@Narretz Am I missing something? And what's the benefit of using a relative link to a hardcoded branch ?

Comment (Narretz) [reply .github/PULL_REQUEST_TEMPLATE.md]: Ah, okay the relative format only works well if you are in the repository view / file tree. It doesn't work as part of the PR template. Just leave it absolute.

**Review by Narretz — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (frederikprijck) [reply .github/PULL_REQUEST_TEMPLATE.md]: I think we can use `/Developers.md`!

**Pull request merged**